Doc/howto/descriptor.rst

```diff
@@ -282,7 +282,7 @@ this::
         . . .
         def __get__(self, obj, objtype=None):
             "Simulate func_descr_get() in Objects/funcobject.c"
-            return types.MethodType(self, obj, objtype)
+            return types.MethodType(self, obj)
```

Comment:
Do you want me to remove that as well? I left it because I think the signature should match ``__get__(self, instance, owner)`` as in regular descriptors right?

btw, I did sign the CLA, please check and let me know if I need to review.